testing/btest/Baseline/scripts.base.protocols.dns.tkey/output

```diff
@@ -0,0 +1,3 @@
+### BTest baseline data generated by btest-diff. Do not edit. Use "btest -U/-u" to update. Requires BTest >= 0.63.
```

Comment:
could you unset  TEST_DIFF_CANONIFIER= for `output`, then inception and expiration timestamps won't be canonified to XXXXXXXXXX.XXXXXX :-)

Also, printing the individual fields explicitly, and using `|ans$key_data|` is easier to grasp.

Mostly FYI, nothing to change: Zeek script has `assert` now, but many of the existing tests aren't using it and we still do a lot of baselines.